bitcoin-ticker/bitcoinTicker.js

```diff
@@ -1,6 +1,32 @@
+const http = require('https');
 
-function bitcoinTicker(...args) {
-  return args;
+const options = {
+  hostname: 'api.coinmarketcap.com',
+  path: '/v2/ticker/',
+  method: 'GET',
+};
+
+function bitcoinTicker({ limit }) {
+  return new Promise((resolve) => {
+    http.get(options, (res) => {
```

Comment:
can we go with a more descriptive name than options?

Reply:
@khuddus Can you give any example?

Reply:
Like requestOptions. Options can be applicable to anything.